lib/pact/init/consumer.rb

```diff
@@ -0,0 +1,43 @@
+require 'erb'
+
+module Pact
+  module Init
+    class Consumer
+
+      def self.run(names = {})
+        new.run(names)
+      end
+
+      def run(names)
+        @names = parse_names(names)
+        create_directory
+        create_files
+        generate_pact_helper
+      end
+
+      def create_directory
+        FileUtils.mkdir_p(provider_dir)
+      end
+
+      def provider_dir
+        'spec/service_providers'
+      end
+
+      def create_files
+        FileUtils.touch(provider_dir+'/'+'pact_helper.rb')
+      end
+
+      def generate_pact_helper
+        template_string = File.read(File.expand_path( '../templates/pact_helper.erb', __FILE__))
+        render = ERB.new(template_string).result(binding)
+        File.open(provider_dir+'/'+'pact_helper.rb', "w+"){ |f| f.write(render) }
+      end
+
+      def parse_names(names)
+        names[:consumer] = 'My Consumer' unless names.has_key? :consumer
```

Comment:
This would be a great place to put the `strip` and `strip.downcase.gsub(' ', '_')` logic. It's much easier to unit test logic when it's in a class than when it's in an erb - views should generally be as dumb as possible.